### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by Immortalin**
For Rust 1.50 and latest version of crate.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -27,11 +27,11 @@ This will allow Cargo to download, build, and cache Windows support as a package
...
-      Windows::Win32::WindowsProgramming::CloseHandle,
-      Windows::Win32::WindowsAndMessaging::MessageBoxA,
-      Windows::Data::Xml::Dom::*,
-      Windows::Win32::SystemServices::{
-          CreateEventW, SetEvent, WaitForSingleObject
+      windows::win32::windows_programming::CloseHandle,
+      windows::win32::windows_and_messaging::{MessageBoxA, HWND, MB_FLAGS},
+      windows::data::xml::dom::*,
+      windows::win32::system_services::{
+          CreateEventW, SetEvent, WaitForSingleObject, PWSTR
```
Comment (tim-weis) [inline readme.md:34]: This is undoing the changes needed as a result of [this pull request](https://github.com/microsoft/windows-rs/pull/646).

File: readme.md [hunk 2]
```diff
@@ -45,12 +45,10 @@ mod bindings {
...
-      Windows::Win32::WindowsProgramming::CloseHandle,
-      Windows::Win32::WindowsAndMessaging::MessageBoxA,
-      Windows::Data::Xml::Dom::*,
-      Windows::Win32::SystemServices::{
-          CreateEventW, SetEvent, WaitForSingleObject
-      },
+    windows::data::xml::dom::*,
+    windows::win32::system_services::{CreateEventW, SetEvent, WaitForSingleObject, PWSTR},
+    windows::win32::windows_and_messaging::{MessageBoxA, HWND, MB_FLAGS},
+    windows::win32::windows_programming::CloseHandle,
```
Comment (tim-weis) [inline readme.md:51]: Same here (see [this pull request](https://github.com/microsoft/windows-rs/pull/646)).

File: readme.md (hunk 1 above)
Comment (Immortalin) [reply readme.md:34]: The current README examples don't work with the latest version on cargo.

Comment (tim-weis) [reply readme.md:34]: Correct. But they do work with the latest version on the *master* branch. This is what's going to go into the next revision, and the README has been updated to reflect that already. That's the reason why all the tests are failing.

Comment (Immortalin) [reply readme.md:34]: Any timeline for a new crate release?

Comment (kennykerr) [reply readme.md:34]: Usually once a week. Probably on Monday.

**Pull request closed without merging**